### PR TITLE
【Fix】Add feature directory

### PR DESCRIPTION
**Pull request by mox692**
ref: [slack](https://tpa-sendai.slack.com/archives/C01DZ8F1P1S/p1606223681134100)

```
├── feature
│  └── question
│    ├── data
│    │  ├── datasources
│    │  │  └── question_service.dart
│    │  ├── models
│    │  │  └── question.dart
│    │  └── repositories
│    │    ├── question_repository.dart
│    │    └── question_repository_impl.dart
│    ├── domain
│    │  └── entities
│    │    └── question.dart
│    └── presentation
│      ├── pages
│      │  ├── app.dart
│      │  ├── edit.dart
│      │  ├── home.dart
│      │  ├── setting.dart
│      │  └── start.dart
│      └── widgets
│        └── appBar.dart
└── main.dart

11 directories, 12 files
```

### CODE REVIEW

**Comment by mox692** [comment]
I chenge other files that is already in project.


```
.
├── core
│   ├── presentation
│   │   ├── common_widgets
│   │   │   └── appBar.dart
│   │   └── themes
│   └── services
├── feature
│   ├── edit
│   │   ├── data
│   │   ├── domain
│   │   │   └── entity
│   │   └── presentation
│   │       ├── pages
│   │       │   └── edit_page.dart
│   │       └── widgets
│   ├── home
│   │   └── presentation
│   │       └── home_page.dart
│   ├── main
│   │   └── presentation
│   │       └── main_page.dart
│   ├── question
│   │   ├── data
│   │   │   ├── datasources
│   │   │   │   └── question_service.dart
│   │   │   ├── models
│   │   │   │   └── question.dart
│   │   │   └── repositories
│   │   │       ├── question_repository.dart
│   │   │       └── question_repository_impl.dart
│   │   ├── domain
│   │   │   └── entities
│   │   │       └── question.dart
│   │   └── presentation
│   │       ├── pages
│   │       │   ├── edit.dart
│   │       │   ├── setting.dart
│   │       │   └── start.dart
│   │       └── widgets
│   │           └── appBar.dart
│   ├── setting
│   │   ├── data
│   │   ├── domain
│   │   │   └── entity
│   │   └── presentation
│   │       ├── pages
│   │       │   └── setting_page.dart
│   │       └── widgets
│   └── start
│       ├── data
│       ├── domain
│       │   └── entity
│       └── presentation
│           ├── pages
│           │   └── start_page.dart
│           └── widgets
└── main.dart

```

File: lib/core/presentation/common_widgets/appBar.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+class MyAppBar extends AppBar {
+  String apptitle = 'Get The Job!';
```
Comment (elvi117) [inline lib/core/presentation/common_widgets/appBar.dart]: it's not an `apptitle`, it's `appBarTitle` or `pageTitle`

please make it as final: 
`final String apptitle;`

and later in the constructor:

`MyAppBar({this.apptitle = 'Get The Job!'})`

File: lib/core/presentation/common_widgets/appBar.dart [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  MyAppBar({this.apptitle})
+      : super(
+          backgroundColor: Colors.teal,
+          title: Text(
+            'Get The Job!',
+            style: TextStyle(fontSize: 20),
+          ),
+        );
+}
```
Comment (elvi117) [inline lib/core/presentation/common_widgets/appBar.dart:14]: run "Reformat code with dartfmt", I guess that something is wrong with style

File: lib/feature/question/presentation/pages/edit.dart [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return Scaffold(
+      appBar: MyAppBar(),
+      body: Container(
+        padding: EdgeInsets.all(32.0),
+        child: Center(
+          child: Column(
+            children: <Widget>[
+              Text('Edit'),
+              RaisedButton(
+                onPressed: () => Navigator.of(context).pop(),
```
Comment (elvi117) [inline lib/feature/question/presentation/pages/edit.dart]: TIP: if you have got equal headers of methods you can push it directly like here:
`onPressed: Navigator.of(context).pop`

Comment (elvi117) [reply lib/feature/question/presentation/pages/edit.dart]: it looks very fancy IMHO 😄

File: lib/feature/question/presentation/pages/setting.dart [hunk 4]
```diff
@@ -74,26 +70,23 @@ class _SettingPageState extends State<SettingPage> {
...
+                          child: Text("ONLY WEEK QUESTION"),
                         ),
                       ),
-                      Switch(value: _values[0],
-                        onChanged: (value){
-                          setState((){
+                      Switch(
+                        value: _values[0],
+                        onChanged: (value) {
+                          setState(() {
```
Comment (elvi117) [inline lib/feature/question/presentation/pages/setting.dart]: in clean architecture, you don't need to use setState any more, whole this stuff should be moved to bloc. Bloc is responsible of any logic in your page. Page should be dummy and be responsible only on widgets and how to display them

File: lib/feature/setting/presentation/pages/setting_page.dart [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+                      Padding(
+                        padding: const EdgeInsets.all(24.0),
+                        child: Container(
+                          child: Text("ONLY WEEK QUESTION"),
+                        ),
+                      ),
+                      Switch(
+                        value: _values[0],
+                        onChanged: (value) {
+                          setState(() {
```
Comment (elvi117) [inline lib/feature/setting/presentation/pages/setting_page.dart]: same as above:

in clean architecture, you don't need to use setState any more, whole this stuff should be moved to the bloc. Bloc is responsible for any logic in your page. The `page` should be a dummy and be responsible only on widgets and how to display them

File: lib/feature/start/presentation/pages/start_page.dart [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+                    color: Colors.amber,
+                    onPressed: () => Navigator.of(context).pop(),
+                    child: Text('Exit'),
+                  ),
+                  SizedBox(width: 50),
+                  RaisedButton(
+                    color: Colors.green,
+                    // onPressed: () => Navigator.of(context).pop(),
+                    onPressed: () => {
+                      setState(
```
Comment (elvi117) [inline lib/feature/start/presentation/pages/start_page.dart]: again:
in clean architecture, you don't need to use setState any more, whole this stuff should be moved to the bloc. Bloc is responsible for any logic in your page. The page should be a dummy and be responsible only on widgets and how to display them

File: lib/feature/start/presentation/pages/start_page.dart [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class BodyContainer extends StatelessWidget {
```
Comment (elvi117) [inline lib/feature/start/presentation/pages/start_page.dart]: please separate any widget to a new file always

File: lib/feature/start/presentation/pages/start_page.dart [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+                    );
+                  },
+                );
+        },
+      ),
+    );
+  }
+}
+
+class QuestionCard extends StatelessWidget {
```
Comment (elvi117) [inline lib/feature/start/presentation/pages/start_page.dart]: same as above

**Review by elvi117 — CHANGES_REQUESTED** [review]

File: lib/feature/question/presentation/pages/edit.dart (hunk 3 above)
Comment (mox692) [reply lib/feature/question/presentation/pages/edit.dart]: i didn't know that!!
In the future, I'll do that way.

File: lib/feature/question/presentation/pages/setting.dart (hunk 4 above)
Comment (mox692) [reply lib/feature/question/presentation/pages/setting.dart]: Oh,,,, I got it.
I've just understand that I can't escape from Bloc!!!!!

I'll see name-game again and learn bloc.😄

File: lib/core/presentation/common_widgets/appBar.dart (hunk 2 above)
Comment (mox692) [reply lib/core/presentation/common_widgets/appBar.dart:14]: My android studio forced this style when i saved file or run `flutter format` 😕

**Comment by mox692** [comment]
@elvi117 
I Fixed reviewed point except for Bloc.
Can I merge to `dev` at this stage and create branch which will implement bloc ?

**Review by elvi117 — APPROVED** [review]

**Pull request merged**